cloudsensei/run.sh

```diff
@@ -0,0 +1,63 @@
+PROJECT_NAME="CloudSensei"
+SUCCESS_OUTPUT_PATH="/dev/null"
+ERROR_LOG="$(mktemp -d)/stderr.log"
+
+
+action="$1"
+
+if [ "$action" = "apply"  ]; then
+   echo "Clearing if previously created zip file"
+    PROJECT_PATH="$PWD/$PROJECT_NAME.zip"
+    if [ -f $PROJECT_PATH ]; then
+        rm -rf  $PROJECT_PATH
+        rm -rf ./package
+        echo "Deleted Previously created zip file"
+    fi
+
+    pip install --target ./package -r requirements.txt > $SUCCESS_OUTPUT_PATH
+    pushd package
+    zip -r ../$PROJECT_NAME.zip . > $SUCCESS_OUTPUT_PATH
+    popd
+    zip -g $PROJECT_NAME.zip lambda_function.py > $SUCCESS_OUTPUT_PATH
+
+  pushd terraform
+  echo "#############################"
+  echo "Creating the lambda lambda_function using terraform"
+  if [ -n "$ACCOUNT_ID" ]; then
```

Comment:
can we use case instead of if-else statement ?

Reply:
Here, i was checking whether account_id is exported or not.